src/SqlUtils.ts

```diff
@@ -60,73 +54,105 @@ export default class SqlUtils {
      */
     private static async tryConnection(config: SqlConnectionConfig, useMaster?: boolean): Promise<ConnectionResult> {
         // Clone the connection config so we can change the database without modifying the original
-        const connectionConfig = JSON.parse(JSON.stringify(config.Config)) as mssql.config;
+        const connectionConfig = JSON.parse(JSON.stringify(config.Config)) as config;
         if (useMaster) {
             connectionConfig.database = "master";
         }
-
+        
+        let sqlCmdError = '';
         try {
             core.debug(`Validating if client has access to '${connectionConfig.database}' on '${connectionConfig.server}'.`);
-            const pool = await mssql.connect(connectionConfig);
+            let sqlCmdCall = this.buildSqlCmdCallWithConnectionInfo(connectionConfig);
+            sqlCmdCall += ` -Q "select getdate()"`;
```

Comment:
I wonder if a user might see this and get confused/concerned.  Maybe `SELECT 'Validating connection from GitHub SQL Action'`?